src/browser/topbar.ts

```diff
@@ -68,11 +68,12 @@ const repaint = () => {
   ctx.stroke()
 }
 const createCanvas = () => {
-  canvas = document.createElement('canvas')
+  const canvas = document.createElement('canvas')
```

Comment:
Question: it used to use `canvas` declared (cached) outside of the function. You're introducing a new `canvas` variable that shadows the global one. Is it intentional? Why?

Reply:
Oops, I missed that. Will rollback `const` definition